src/main/kotlin/app/revanced/patches/tiktok/interaction/speed/PlaybackSpeedPatch.kt

```diff
@@ -13,8 +13,8 @@ import com.android.tools.smali.dexlib2.Opcode
     name = "Playback speed",
     description = "Enables the playback speed option for all videos.",
     compatiblePackages = [
-        CompatiblePackage("com.ss.android.ugc.trill"),
-        CompatiblePackage("com.zhiliaoapp.musically")
+        CompatiblePackage("com.ss.android.ugc.trill", ["30.8.4"]),
+        CompatiblePackage("com.zhiliaoapp.musically", ["30.8.4"])
```

Comment:
It's not even working on 30.8.4
https://github.com/ReVanced/revanced-patches/issues/2814

Reply:
Do you know the lastest version which that patch still work?

Reply:
Maybe 30.7.xx
But it's too old.
TikTok always shows update popup on this version.

Reply:
nvm, I bumped that patch.